src/main/resources/com/mesosphere/velocity/marathon/MarathonRecorder/help-credentialsId.html

```diff
@@ -1,7 +1,36 @@
 <div>
-    <p>
-        To use a DC/OS <a href="https://docs.mesosphere.com/administration/security-and-authentication/auth-api/">authentication token</a>, select a credential of type secret text, where the text is a token authorized to access this Marathon instance.
-        <br />
-        To authenticate with HTTP basic authentication, select a credential of type username with password.
-    </p>
+    <p>There are two types of authentication supported:</p>
+    <ul>
+        <li>Basic Authentication (Username and Password)</li>
+        <li>DC/OS Service Account (String)</li>
+    </ul>
+    <h3>Basic Authentication</h3>
+    <p>To authenticate with HTTP basic authentication, select a credential of type username with password. <strong>This
+        authentication type does not support DC/OS.</strong></p>
```

Comment:
Hm, might need to be the other way around? `DC/OS does not support this authentication type.` 

is it worth mentioning that this is typically used for open source Marathon deployments?